mutornadomon/config.py

```diff
@@ -28,4 +30,7 @@ def initialize_mutornadomon(tornado_app=None, publisher=None, publish_interval=N
         web_collector = WebCollector(monitor, tornado_app)
         web_collector.start()
 
+    utilization_collector = UtilizationCollector(monitor, publish_interval)
```

Comment:
We should probably implement the flushing of the counter on gauge to run on the same `PeriodicCallback` as the publisher. Otherwise there can be discrepancies between where we flush the metric between the moments of publishing.